core/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -1144,6 +1098,10 @@ public static MetaData fromXContent(XContentParser parser) throws IOException {
                         while ((token = parser.nextToken()) != XContentParser.Token.END_OBJECT) {
                             builder.put(IndexTemplateMetaData.Builder.fromXContent(parser, parser.currentName()));
                         }
+                    } else if ("warmers".equals(currentFieldName)) {
+                        // TODO: remove this condition in 4.0
```

Comment:
just our of curiosity do we have a bwc index that has warmers? I think we should add one if we don't or regenerate the BWC indices with warmers?